packages/front-end/components/EventWebHooks/EventWebHookDetail/EventWebHookDetail.tsx

```diff
@@ -135,17 +136,34 @@ export const EventWebHookDetail: FC<EventWebHookDetailProps> = ({
           </div>
         </div>
 
-        <div className="d-flex align-items-center mt-2">
-          <span className="text-muted ml-1 mr-2" style={{ fontSize: "1rem" }}>
-            <TbWebhook className="d-block" />
-          </span>
-          <span className="font-weight-bold">&nbsp;Events</span>
-          <div className="flex-grow-1 d-flex flex-wrap ml-3">
-            {events.map((eventName) => (
-              <span key={eventName} className="mr-2 badge badge-purple">
-                {eventName}
+        <div className="row">
+          <div className="col-xs-12 col-md-6">
+            <div className="d-flex align-items-center mt-2">
+              <span
+                className="text-muted ml-1 mr-2"
+                style={{ fontSize: "1rem" }}
+              >
+                <TbWebhook className="d-block" />
               </span>
-            ))}
+              <span className="font-weight-bold">&nbsp;Events</span>
+              <div className="flex-grow-1 d-flex flex-wrap ml-3">
+                {events.map((eventName) => (
+                  <span key={eventName} className="mr-2 badge badge-purple">
+                    {eventName}
+                  </span>
+                ))}
+              </div>
+            </div>
+          </div>
+
+          <div className="col-xs-12 col-md-6">
+            <div className="mt-2">
+              {eventWebHook.enabled ? (
+                <Tag tag="Webhook enabled" color="#28a745" />
```

Comment:
I think it would be better to use `<Badge>` or even `<span className="mr-2 badge">`, as tags imply tag functionality (search, etc).

Reply:
Thanks! Yes, that does look like a better component. Just realized `<Tag />` is potentially making API calls via `useDefinitions()` too. Will make the change.

Reply:
FYI @bryce-fitzsimons badge change pushed up.

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/113377031/225407491-d29c72f6-ffab-4ee0-b4a9-3a0b34536b68.png">

<img width="1003" alt="image" src="https://user-images.githubusercontent.com/113377031/225407532-b03609b5-126d-430f-8a8e-a3f98009f4d8.png">